### PR TITLE
fix: the test parts set value to new Date() when there is no prop

### PR DESCRIPTION
**Pull request by oh3vci**
## Description

I thought I fixed all of the new Date operator related DatePicker test issue. #340
However, there are some parts of set value to new Date when there is no prop.
So, I modified them.

## Related Issues

#337 

## Tests

jest

## Checklist

- [x] I read the [Contributor Guide](https://github.com/dooboolab/dooboo-ui/blob/master/CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
- [x] Run `yarn test` or `yarn test -u` if you need to update snapshot.
- [x] Run `yarn lint`
- [x] I am willing to follow-up on review comments in a timely manner.

### CODE REVIEW

**Comment by JeffGuKang** [comment]
I am not sure this PR can cover `toLocaleString` issue. It can be run differently on each OS locale without mocking.

**Comment by oh3vci** [comment]
> I am not sure this PR can cover `toLocaleString` issue. It can be run differently on each OS locale without mocking.

What is the **toLocaleString** of `/test/jestSetup.ts`? I think it is for this situation.
Is it not worked? @hyochan

**Comment by oh3vci** [comment]
I thought it was necessary to add a locale prop to solve the locale problem.

**Comment by JeffGuKang** [comment]
@oh3vci 
There is some problems to mock `Date` on React Native environment. I guess it is related to `Babel` but not sure. 
Anyway, it will be good to exclude snapshots from the `Datepicker` test.
https://github.com/dooboolab/dooboo-ui/issues/337#issuecomment-692028016

Checking rendering and branches with props is enough without mocking `Date`.

**Comment by oh3vci** [comment]
I rebased it.

File: main/DatePicker/Calendar/CalendarWeekDays.tsx [hunk 1]
```diff
@@ -39,7 +40,7 @@ function CalendarWeekDays(props: Props): React.ReactElement {
 
   for (let idx = 0; idx <= 6; idx++) {
     const matchMonth = new Date(2020, 8, 6 + idx);
-    const weekDay = matchMonth.toLocaleString('default', {
+    const weekDay = matchMonth.toLocaleString(props.locale, {
```
Comment (JeffGuKang) [inline main/DatePicker/Calendar/CalendarWeekDays.tsx:43]: Is it OK if props.locale is undefined?

File: main/DatePicker/PickerCalendar.tsx [hunk 2]
```diff
@@ -36,7 +37,7 @@ interface Props {
 }
 
 const PickerCalendar: FC<Props> = (props) => {
-  const { calendarWidth = 300 } = props;
+  const { calendarWidth = 300, locale = 'default' } = props;
```
Comment (oh3vci) [inline main/DatePicker/PickerCalendar.tsx:40]: The default value of locale prop is **default**. It is specified to PickerCalendar that is parent component of CalendarWeekDays.
@JeffGuKang

**Pull request merged**